src/plugins/whiteboard/whiteboard-toolbar.js

```diff
@@ -0,0 +1,312 @@
+import PropTypes from 'prop-types';
+import LineIcon from './icons/line-icon.js';
+import TextIcon from './icons/text-icon.js';
+import ShapeIcon from './icons/shape-icon.js';
+import ArrowIcon from './icons/arrow-icon.js';
+import ResetIcon from './icons/reset-icon.js';
+import { useTranslation } from 'react-i18next';
+import CircleIcon from './icons/circle-icon.js';
+import SelectIcon from './icons/select-icon.js';
+import EraserIcon from './icons/eraser-icon.js';
+import React, { Fragment, useState } from 'react';
+import TriangleIcon from './icons/triangle-icon.js';
+import FontSizeIcon from './icons/font-size-icon.js';
+import FreeDrawIcon from './icons/free-draw-icon.js';
+import RectangleIcon from './icons/rectangle-icon.js';
+import FillColorIcon from './icons/fill-color-icon.js';
+import StrokeWidthIcon from './icons/stroke-width-icon.js';
+import StrokeColorIcon from './icons/stroke-color-icon.js';
+import SwatchesPickerNs from 'react-color/lib/Swatches.js';
+import { Button, Dropdown, Popover, Radio, Tooltip } from 'antd';
+import { DEFAULT_COLOR_SWATCHES, DEFAULT_COLOR_PICKER_WIDTH } from '../../domain/constants.js';
+
+const RadioGroup = Radio.Group;
+const RadioButton = Radio.Button;
+const SwatchesPicker = SwatchesPickerNs.default || SwatchesPickerNs;
+
+export const MODES = {
+  select: 'select',
+  freeDraw: 'freeDraw'
+};
+
+export const FONT_SIZES = {
+  small: 16,
+  medium: 22,
+  large: 26
+};
+
+export const STROKE_WIDTHS = {
+  small: 2,
+  medium: 3,
+  large: 6
+};
+
+const SHAPE_TYPES = {
+  line: 'line',
+  arrow: 'arrow',
+  circle: 'circle',
+  triangle: 'triangle',
+  rectangle: 'rectangle'
+};
+
+const MENUS = {
+  shape: 'shape',
+  fontSize: 'fontSize',
+  strokeWidth: 'strokeWidth'
+};
```

Comment:
We usually (at least in most places, I guess) have singular names for enum objects. We could rename them to `MODE`, `FONT_SIZE` etc. (though I am not very strongly opinionated on that). Up to you.

Reply:
Renamed them to singular, I don't know why I made them plural to begin with.